mobile-workspace/latest/config/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configure Mobile Workspace
+---
+
+Find Alfresco Mobile Workspace at [Google Play](https://play.google.com/store/apps/details?id=com.alfresco.content.app&hl=en-GB&ah=tbrTPT50zRyrvFT0xxYx0IRH6DA){:target="_blank"} for Android or [Apple App Store](https://apps.apple.com/gb/app/alfresco-mobile-workspace/id1514434480){:target="_blank"} for iOS.
+
+## First screen
```

Comment:
Can we rename this to Login screen or Landing screen?

Reply:
done